### ISSUE TITLE
MovieTexture plugin support?

### ISSUE THREAD

**Issue by Flenper** [nesrak1/UABEA#240]
Is it possible to add support for a plugin to export/import MovieTexture files? It was in UABE v2.2, but then it was removed in future versions for an unknown reason.

**Comment by nesrak1** [comment]
It wasn't on the plan for the foreseeable future. I don't know how they are stored, but it may not be terrible if it's just a list of texture2ds or an avi or something. I don't think I actually have any games that use MovieTexture though.

**Comment by Flenper** [comment]
Yeah, this is the first time I've seen a game that uses MovieTexture. I believe m_MovieData contains the video file itself in ogv format, which needs to be imported/exported.
